Categories/Morphism/Reasoning/Core.agda

```diff
@@ -41,9 +41,12 @@ module Basic where
 open Basic public
 
 module Utils where
-  assoc² : ((i ∘ h) ∘ g) ∘ f ≈ i ∘ (h ∘ (g ∘ f))
+  assoc² : ((i ∘ h) ∘ g) ∘ f ≈ i ∘ h ∘ g ∘ f
```

Comment:
I'm not sure this should be changed. I actually like the more explicit version with parentheses better.